boot/bootutil/src/loader.c

```diff
@@ -919,6 +919,20 @@ boot_erase_region(const struct flash_area *fap, uint32_t off, uint32_t sz)
 }
 
 #if !defined(MCUBOOT_DIRECT_XIP) && !defined(MCUBOOT_RAM_LOAD)
+
+#if defined(MCUBOOT_ENC_IMAGES) || defined(MCUBOOT_SWAP_SAVE_ENCTLV)
+/* Replacement for memset(p, 0, sizeof(*p) that should not get
```

Comment:
```suggestion
/* Replacement for memset(p, 0, sizeof(*p) that does not get
```

Let's make a stronger statement unless you have reason to believe it can be optimized out.